package.json

```diff
@@ -8,17 +8,12 @@
     "@emotion/sheet": "1.0.1",
     "@emotion/utils": "1.0.0",
     "moment": "2.29.1",
-    "ssri": "8.0.1",
-    "hosted-git-info": "3.0.8",
-    "postcss": "8.3.0",
     "browserslist": "4.16.5",
-    "multicast-dns": "7.2.3",
     "web-vitals": "1.1.1",
-    "ws": "7.4.6",
-    "trim-newlines": "3.0.1",
```

Comment:
I guess these are the various pinned dependencies we had to avoid security issues that are no longer needed?

Reply:
Yep :)